doc/source/examples.rst

```diff
@@ -166,14 +167,14 @@ methods
     ...
```

Comment:
Line `import pyndl` might not be necessary

Reply:
I think you are right on that. I deleted the line